tech/post/How to use manila image.md

```diff
@@ -0,0 +1,148 @@
+
+
+How to use manila image in devstack pike version
```

Comment:
Assume you are writing a markdown file, please try to write in a [markdown style](https://guides.github.com/features/mastering-markdown/).